utils/deepsize/Cargo.toml.orig

```diff
@@ -0,0 +1,27 @@
+[package]
```

Comment:
why do we put this inside nearcore?

Reply:
In Rust you cannot declare theirs for external packages. By having deepsize here, we are able to do that.
I'll remove this crate for now and write a new crate from scratch.